### PR TITLE
Always redirect `/__swagger_/` to `/__docs__/`

### LINKED ISSUE #693: Always redirect __swagger__ to __docs__
For legacy support, `/__swagger__/` should always redirect to `/__docs__/`, even when Swagger isn't the selected interface. This is particularly important for RStudio Connect, which currently listens to `/__swagger__/` regardless of the chosen interface.

### PR DESCRIPTION
**Pull request by meztez**
Fix #693 (remove is_swagger condition)

PR task list:
- [x] Update NEWS
- [ ] Add tests
- [ ] Update documentation with `devtools::document()`

### CODE REVIEW

File: R/ui.R [hunk 1]
```diff
@@ -214,11 +212,9 @@ register_docs <- function(name, index, static = NULL) {
     pr$mount(docs_root, docs_router)
 
     # add legacy swagger redirects
```
Comment (schloerke) [inline R/ui.R:214]:
```suggestion
    # add legacy swagger redirects (RStudio Connect)
```

File: NEWS.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ plumber 1.0.0.9999 Development version
 
 * When calling `Plumber$handle()` and defining a new `PlumberEndpoint`, `...` will be checked for invalid names #677  
 
+* `/__swagger__/` now always redirect to `/__docs__/`, even when Swagger isn't the selected interface.
```
Comment (schloerke) [inline NEWS.md:8]:
```suggestion
* `/__swagger__/` now always redirect to `/__docs__/`, even when Swagger isn't the selected interface. (@blairj09 #694)
```

**Review by schloerke — APPROVED** [review]

**Comment by cpsievert** [comment]
Let's expose an option to avoid adding the redirect if people really want that

**Comment by cpsievert** [comment]
Also add a check for an existing route at the `__swagger__` redirect locations

**Review by cpsievert — APPROVED** [review]
LGTM with pending suggestions

**Comment by meztez** [comment]
> Also add a check for an existing route at the `__swagger__` redirect locations

I think only having the options to disable redirects should be ok.
I you want to use those route, use the options to disable redirects?

**Comment by meztez** [comment]
@meztez Split fix to #701 to a separate PR

**Comment by schloerke** [comment]
Thank you @meztez for splitting the PR!

**Comment by schloerke** [comment]
Closing in favor of #704

**Pull request closed without merging**